src/components/EmailForm.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import addToMailchimp from 'gatsby-plugin-mailchimp'
+import './EmailForm.css'
+
+export default class EmailForm extends React.Component {
+  state = {
+    email: ''
+  }
+
+  _handleSubmit = e => {
+    e.preventDefault() // prevent the page from refreshing
```

Comment:
Remove comments, I usually only use comments when writing code that's hard to follow. This isn't too bad 😁